environment.yml

```diff
@@ -19,6 +19,7 @@ dependencies:
   - raimis::pytorch-geometric==2.0.2
   - pytorch-lightning==1.3.8
   - setuptools==59.5.0 # Pin due to https://github.com/pytorch/pytorch/pull/69904
+  - torchmetrics <0.8
```

Comment:
Why are you adding torchmetrics here?

Reply:
It is a dependency of PyTorch Lightning, but the latest versions (0.8) is no longer compatible with the old PyL, which we are using.